denops/@ddu-uis/ff.ts

```diff
@@ -959,7 +883,7 @@ export class Ui extends BaseUi<Params> {
       await this.#saveCursor(args.denops, bufnr, cursorPos[1]);
 
       // Change real cursor
-      await args.denops.call("ddu#ui#ff#_cursor", cursorPos[1], 0);
+      await fn.call(args.denops, cursorPos[1], 0);
```

Comment:
Occurs `Error: Failed to call call(2, 0): Vim:E714: List required (code: 0)`.
Maybe `fn.cursor`?